wase/type.flow

```diff
@@ -727,7 +727,7 @@ typecheckWaseInstruction(acc : DslTypeAcc, op : string, pars : [DslAst], opargs
 			if (i == nargs - 1) i32 else infer();
 		}));
 		acc.unifyTypes(acc, opargs, argTypes);
-		infer();
+		if (false) list0 else infer();
```

Comment:
I guess this is infer?

Reply:
infer = DSLNode("") - unknown type. but for void, the result must be list0